### PR TITLE
Add the latest hotfixes to develop

### PR DESCRIPTION
**Pull request by smesseim**
The hotfixes to fix tooling and testing need to be merged into develop.

### CODE REVIEW

**Comment by RaviShivam** [comment]
The bubble threshold is **not** a fixed constant. This should be determined by (zoom)level the user is in. Is it really necessary to define it as a class constant?

**Comment by smesseim** [comment]
For now it is, because the client does not request a specific zoom level (yet). Once the client requests a specific zoom level, the constant can be eliminated.

File: src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java [hunk 1]
```diff
@@ -1,35 +1,41 @@
...
+import java.util.Set;
 
+/**
+ * A bubble represents a higher level node. A bubble can contain multiple segments, or multiple
+ * bubbles.
+ */
 public class Bubble implements Node {
     private int id;
     private int x;
     private int y;
```
Comment (RaviShivam) [inline src/main/java/io/github/programminglife2016/pl1_2016/collapser/Bubble.java]: Why is the field `isBubble` removed?

**Comment by RaviShivam** [comment]
Okay. LGTM

**Pull request merged**